### PR TITLE
Prepare to distribute the tool on PyPI

### PR DESCRIPTION
**Pull request by siddharth-krishna**

### CODE REVIEW

**Comment by siddharth-krishna** [comment]
Regression tests are failing because the name of the tool changed, so it can't find `xl2times` in the main branch to compare to. Perhaps I can compare the numbers manually and override the check for this PR when we're ready?

File: pyrightconfig.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "include": [
-    "times_reader",
+    "xl2times",
     "utils",
     "times_excel_reader.py"
```
Comment (olejandro) [inline pyrightconfig.json:5]: Should this one be renamed as well?

**Comment by olejandro** [comment]
@siddharth-krishna I've compared the numbers manually and it looks okay. I'll merge this so that we can move on.

**Review by olejandro — APPROVED** [review]

**Comment by siddharth-krishna** [comment]
Give me a few minutes

File: pyrightconfig.json (hunk 1 above)
Comment (siddharth-krishna) [reply pyrightconfig.json:5]: Ah, we should remove that. It's a remnant of the previous folder structure, before we started using `pyproject.toml` and `setuptools`

**Comment by olejandro** [comment]
Sorry, was too fast. Did not know you were on it!

**Pull request merged**